db/migrate/20240228140011_update_iseq_flowcell_entity_type_comment.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateIseqFlowcellEntityTypeComment < ActiveRecord::Migration[7.0]
+  def change
+    change_column_comment :iseq_flowcell, :entity_type, "Lane type: library, library_control, library_indexed, library_indexed_spike."
```

Comment:
This is perfectly fine to do what the issue requests.  I would suggest, though, implementing a `def up` and a `def down` method in this particular migration so that it's possible to rollback the migration if needed.  What you've written here would be in `def up` and `def down` would change the comment back to what it was before the migration.

Reply:
This might be overkill.  I just don't like unrevertable migrations because they lose information.

Reply:
yeah I agree. I'll update the migration.

Reply:
Perhaps you already know, but in case not: `def change` was introduced later than the `up` and `down` methods and it works well for some operations that are only adding new things.  Like if you add a column in a `change` method, the migration already knows that to go back it needs to remove that column.  It doesn't keep track of what it's doing when it changes a comment, a type or removes a column, so it has no way of going back in those cases.